### PR TITLE
link to numpy.org/doc for numpy docs

### PR DESCRIPTION
**Pull request by mattip**
update that newer documentation version are on numpy.org/doc

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -52,6 +48,7 @@ Welcome! This is the documentation for Numpy and Scipy.
   <ul class="thumbnails" style="margin-left: 5%;">
   <li class="span6">
   <div class="pull-left">
+      <p>For newer versions see https://www.numpy.org/doc</p>
```
Comment (larsoner) [inline index.rst]:
```suggestion
      <p>For newer versions see <a href="https://www.numpy.org/doc" class="extlink">https://www.numpy.org/doc</a></p>
```

**Review by larsoner — COMMENTED** [review]
It looks like the SciPy set is a superset of the NumPy set:

https://docs.scipy.org/doc/

So I'm not clear on what this actually adds.

But if we do want it, it should probably be in a proper `<a` tag to make a link.

File: index.rst (hunk 1 above)
Comment (mattip) [reply index.rst]: fixed

**Comment by mattip** [comment]
NumPy is intending to host its own documentation going forward at numpy.org/doc, future versions will only appear there and not on this server.

**Review by larsoner — APPROVED** [review]
Okay in that case, sounds reasonable to me!

**Comment by larsoner** [comment]
Thanks @mattip

**Comment by mattip** [comment]
Thanks, page regenerated and uploaded.

**Pull request merged**